src/__swaps__/screens/Swap/hooks/useSwapEstimatedGasLimit.ts

```diff
@@ -98,8 +84,18 @@ export function useSwapEstimatedGasLimit(
       cacheTime: 60 * 1000, // 1min
       notifyOnChangeProps: ['data', 'isFetching'],
       keepPreviousData: true,
-      placeholderData: gasUnits.basic_swap[chainId],
+      enabled: !!chainId && !!quote && !!assetToSell && assetToSell.chainId === chainId,
+      initialData,
+      initialDataUpdatedAt: 0,
```

Comment:
can you help me understand why you have `initialDataUpdatedAt` like this here?

Reply:
since I'm setting a initialData it is considered fresh so won't refetch until the stale time passes, setting it to zero makes it instantly start refetching the query